### PR TITLE
Desktop scaling for trips and profile pages

### PR DESCRIPTION
**Pull request by ang-zy**
Forces the trips and profile pages to display as mobile on large screens

### CODE REVIEW

File: src/client/styles/components/_trip.scss [hunk 1]
```diff
@@ -27,8 +27,8 @@
     margin-right: 5%;
 
     img {
-      width: 22vw;
-      height: 22vw;
+      width: 100%;
+      height: 100%;
```
Comment (karenfrilya97) [inline src/client/styles/components/_trip.scss]: will this cause the image ratio to not be 1:1?

Comment (ang-zy) [reply src/client/styles/components/_trip.scss]: It appeared to be 1:1 on local testing

**Comment by ang-zy** [comment]
Closed due to excessive merge conflicts

**Comment by ang-zy** [comment]
Superseded by #218

**Pull request closed without merging**